.github/workflows/build-test-distribute.yml

```diff
@@ -349,6 +349,16 @@ jobs:
           pattern: RunnerSysStats*
           merge-multiple: true
 
+      - uses: aws-actions/configure-aws-credentials@v4
+        id: aws-creds
+        with:
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_CI_STAT_API_ROLE_ARN }}
+          output-credentials: true
```

Comment:
this should be under "internal-build" if

Reply:
yeah, it already exists at the job level (line 340):
`if: ${{ always() && needs.config.outputs.internal_build == 'true' }}`